src/kalman/BallFilter.h

```diff
@@ -0,0 +1,71 @@
+//
+// Created by rolf on 17-11-19.
+//
+
+#ifndef RTT_BALLFILTER_H
+#define RTT_BALLFILTER_H
+
+#include <roboteam_proto/messages_robocup_ssl_detection.pb.h>
+#include <roboteam_proto/WorldBall.pb.h>
+
+#include <utility>
+#include "util/KalmanFilter.h"
+#include "CameraFilter.h"
+
+class BallFilter : public CameraFilter {
+    typedef KalmanFilter<4, 2> Kalman;
```

Comment:
this typedef makes things actually harder to read

Reply:
How so?If I do not use the typedef,  I would  constantly need to type KalmanFilter<4,2>::Vector to create and set the vectors in the KalmanFilter in the CPP file. I think it's cleaner to use a typedef throughout the file instead of being confronted with magic numbers every time I need to do something related to the filter.